protocol/synthetix/contracts/storage/Vault.sol

```diff
@@ -144,6 +144,40 @@ library Vault {
         }
     }
 
+    /**
+     * @dev Traverses available rewards for this vault and the reward id, and returns an accounts
+     * pending claim on them according to the amount of debt shares they have.
+     */
+    function getReward(
+        Data storage self,
+        uint128 accountId,
+        bytes32 rewardId
+    ) internal view returns (uint256) {
+        uint256 totalSharesD18 = currentEpoch(self).accountsDebtDistribution.totalSharesD18;
+        uint256 actorSharesD18 = currentEpoch(self).accountsDebtDistribution.getActorShares(
+            accountId.toBytes32()
+        );
+
+        RewardDistribution.Data storage dist = self.rewards[rewardId];
+
+        if (address(dist.distributor) == address(0)) {
+            revert RewardDistributorNotFound();
+        }
+
+        uint256 currentPending = dist.claimStatus[accountId].pendingSendD18;
+        uint256 currentRewardPerShare = dist.rewardPerShareD18;
+
+        currentRewardPerShare += dist.getEntry(totalSharesD18).toUint().to128();
+
+        currentPending +=
+            currentPending +
```

Comment:
This looked a little weird to me, we're doing `+=` here and adding `currentPending`, a bit unusual so just double checking that its on purpose. Its basically the same as doing:
```
currentPending = (currentPending * 2)  + actorSharesD18.mulDecimal(
   currentRewardPerShare - dist.claimStatus[accountId].lastRewardPerShareD18
)`
```
Right? My mutation skills lacking...
It is well tested so I assume it's right though...

Reply:
Yeah this was copypasta lol so let me test